### DISCUSSION TITLE
Adding more collaborators

### DISCUSSION THREAD

**Discussion by stephenberry** [stephenberry/glaze#456] [General]
I think it would be great to add more collaborators to glaze. So, I've thrown up this discussion so that I can tag those who've made helpful contributions and ask if you'd like to be added as a collaborator on GitHub.

Being added as a collaborator will allow you to:
- Help merge pull requests from others
- Merge into main and create new releases (important for hot fixes)
- Help moderate discussions and issues
- etc.

I'm still going to be heavily invested in leading glaze development. I just want to allow others to have more control to make development more efficient and better support the library as it grows.

**Comment by stephenberry** [comment]
@jbbjarnason, would you be interested in being added as a collaborator?

**Reply by jbbjarnason** [reply]
Yes, it would be well appreciated. We are heavily invested in the library so going forward it would be beneficial for both parties I think. 
Great job so far!

**Reply by stephenberry** [reply]
Thanks! I just added you. It's great to have developers using and supporting the library in various contexts.

**Comment by stephenberry** [comment]
@justend29, would you be interested in being added as a collaborator?

**Reply by justend29** [reply]
Most definitely! I actually have a backlog of Glaze additions and tests that I would like to add to Glaze in the future, so I have an ongoing interest in the library.
Nevertheless, I still think your leadership is invaluable.

**Reply by stephenberry** [reply]
Great! I just added you. Thanks for your help!
